src/components/newsItem.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import { Link } from "gatsby"
+import "./newsItem.scss"
+
+export default function NewsItem({
+  image,
+  date,
+  title,
+  description,
+  id,
+  hoverEffects,
+}) {
+  return (
+    <div className={hoverEffects ? "news-item news-item--hover" : "news-item"}>
+      <div className="news-item__back-panel"></div>
+      <div className="news-item__wrapper">
+        <div className="news-item__image-container">
+          {image && (
+            <div
+              className="news-item__image"
+              style={{
+                backgroundImage: `url("${image}")`,
+              }}
+            ></div>
+          )}
+        </div>
+        {date && <h5>{date}</h5>}
```

Comment:
The previous developer has just completely messed up the use of `h` tags.

Can we go ahead and switch this out to use the `<date>` tag instead? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time